dbus_service.py

```diff
@@ -129,6 +129,14 @@ def __init__(
         gobject.timeout_add(self._get_sign_of_life_interval() * 60 * 1000, self._sign_of_life)
 
     ## read config file
+    def stringToBool(self, s):
```

Comment:
in helpers.py, we have `def is_true(val)...`. Please use this one, if there's no reason of not using it.
I propose to change its implementation to `return val in (1, '1', True, "True", "true")`.